### PR TITLE
[Editor]Removed materialize.css, use MUI for toolbar and dialog

### PR DESCRIPTION
**Pull request by AhmetTanakol**

### CODE REVIEW

File: packages/editor/example/app-bar/EditorBar.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import FileDownload from '@material-ui/icons/FileDownload';
+import FolderOpen from '@material-ui/icons/FolderOpen';
+import ImportExport from '@material-ui/icons/ImportExport';
+import ModelSchemaDialog from './dialogs/ModelSchemaDialog';
+import { Actions, getData, getSchema } from '@jsonforms/core';
+
+const ajv = new AJV({allErrors: true, verbose: true});
```
Comment (edgarmueller) [inline packages/editor/example/app-bar/EditorBar.tsx]: Use `createAjv` to have a unified configuration of the validator

File: packages/editor/example/app-bar/EditorBar.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  exportDialog: {
+    open: boolean
+  };
+}
+
+class EditorBar extends
+  React.Component<EditorBarProps & WithStyles<'root' | 'flex' | 'rightIcon' | 'button'>,
+                  EditorBarState> {
+  componentWillMount() {
+    this.setState({
```
Comment (edgarmueller) [inline packages/editor/example/app-bar/EditorBar.tsx]: I'd rather initialize state in the constructor

File: packages/editor/example/app-bar/dialogs/ModelSchemaDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  onClose: any;
+  readOnly: boolean;
+  open: boolean;
+  fullScreen?: boolean;
+  rootData?: any;
+}
+
+class ModelSchemaDialog extends
+  React.Component<ModelSchemaDialogProps & WithStyles<'textarea'>, {}> {
+  private textInput;
```
Comment (edgarmueller) [inline packages/editor/example/app-bar/dialogs/ModelSchemaDialog.tsx]: Use `React.createRef` the ref

File: packages/editor/package.json [hunk 4]
```diff
@@ -55,13 +55,21 @@
     "@jsonforms/core": "^2.0.1",
     "@jsonforms/react": "^2.0.1",
     "@jsonforms/webcomponent": "^2.0.1",
+    "@material-ui/core": "1.0.0",
+    "@material-ui/icons": "1.0.0",
+    "@types/lodash": "^4.14.104",
```
Comment (edgarmueller) [inline packages/editor/package.json]: Types are dev dependencies

File: packages/editor/example/app-bar/dialogs/ModelSchemaDialog.tsx (hunk 3 above)
Comment (AhmetTanakol) [reply packages/editor/example/app-bar/dialogs/ModelSchemaDialog.tsx]: We might need to update material-ui in material workspace as well. https://github.com/mui-org/material-ui/issues/11709

**Comment by AhmetTanakol** [comment]
updated

**Comment by edgarmueller** [comment]
@AhmetTanakol Needs to be rebased once more

**Comment by AhmetTanakol** [comment]
Updated

**Comment by AhmetTanakol** [comment]
I updated @types/react version in order to use `createRef` for MUI components. New MUI version depends on the latest @types/react version

**Pull request merged**